extensionsGallery.json

```diff
@@ -3074,7 +3074,47 @@
 					],
 					"statistics": [],
 					"flags": ""
-				}
+				},
+                {
+                  "extensionId": "62",
```

Comment:
Please also update the TotalCount value at the bottom of this file

Reply:
Updated